sql/testdata/system

```diff
@@ -42,6 +42,12 @@ SELECT id FROM system.descriptor
 5
 1000
 
+# Verify we can read "protobuf" columns.
+query I
+SELECT length(descriptor) * (id - 1) FROM system.descriptor WHERE id = 1
```

Comment:
I think I'd rather this test was in cli_util_test since the output there is more useful